<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>셀</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>값</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputbtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>75</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>출력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputbtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>136</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>불러오기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cell_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="value_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>22</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="out_cell_edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>셀</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>값</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="out_value_edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>양재참숯체B</family>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
